onmt/utils/ExtendedCmdLine.lua

```diff
@@ -0,0 +1,325 @@
+------------------------------------------------------------------------------------------------------------------
```

Comment:
maybe we can reduce the length, but I wanted to separate local functions that ruin a bit the visibility?